financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkstepupverification/LinkStepUpVerificationViewModel.kt

```diff
@@ -148,7 +151,7 @@ internal class LinkStepUpVerificationViewModel @Inject constructor(
         updateLocalManifest { it.copy(activeInstitution = activeInstitution.data.firstOrNull()) }
         // Updates cached accounts with the one selected.
         updateCachedAccounts { listOf(selectedAccount) }
-        goNext(Pane.SUCCESS)
+        navigationManager.navigate(NavigateToRoute(success))
         Unit
```

Comment:
Nit: Is this unnecessary now?

Reply:
https://github.com/stripe/stripe-android/pull/6968/commits/c4d9d369ca742c47e133ce7ece1d0b154712c2eb